default.nix

```
@@ -19,7 +19,9 @@ in
   opaline = (super.opaline.override {
     inherit (self) ocamlPackages;
   });
-  esy = callPackage ./ocaml/esy { };
+  inherit esy;
+
+  esy-fhs = callPackage ./ocaml/esy/fhs.nix { inherit esy; inherit (self) buildFHSUserEnv; };
```

```suggestion
  esy-fhs = callPackage ./ocaml/esy/fhs.nix { inherit (self) buildFHSUserEnv esy; };
```

If you apply this suggestion you can revert the diff above